core/docker-compose.yml

```diff
@@ -17,8 +17,10 @@ services:
     volumes:
       - "./traefik.toml:/traefik.toml"
       - "./acme.json:/acme.json"
+      - "./usersfile:/usersfile"
       - "/var/run/docker.sock:/var/run/docker.sock:ro"
     labels:
       - "traefik.http.middlewares.csrfHeader.headers.customrequestheaders.X-Forwarded-Proto=https"
       - "traefik.http.middlewares.csrfHeader.headers.customrequestheaders.X-Forwarded-Port=443"
       - "traefik.http.middlewares.csrfHeader.headers.customrequestheaders.X-Forwarded-Ssl=on"
+      - "traefik.http.middlewares.bauth.basicauth.usersfile=/usersfile"
```

Comment:
Can you confirm this works as `/usersfile` and not `./usersfile`?

Reply:
I think we confirmed it?

Reply:
I'm not confident this is solved, e.g. it does not work on the server with `/usersfile` and at the moment we're not using the usersfile as it didn't work ... I have also tried `./usersfile` and `/complete/path/to/usersfile`.